frontend/public/style/_common.scss

```diff
@@ -403,29 +405,40 @@
   max-width: 600px;
 }
 
-.co-m-modal-link {
+// Enable word-break and append pficon-var-edit icon ::after so that it doesn't wrap without text
+.co-modal-btn-link {
   cursor: pointer;
```

Comment:
@sg00dwin fyi, i still see this line... it's not needed anymore

Reply:
ok, removed this time